io/untrusted/src/lib.rs

```diff
@@ -23,7 +23,7 @@ pub type WriteAll = dyn Fn(&[u8]);
 ///
 /// # Arguments
 /// * `write_all` - The function to use for writing stderr from the enclave
-pub fn stderr_write_all(write_all: &'static WriteAll) {
+pub fn stderr_sink(write_all: &'static WriteAll) {
```

Comment:
TL;DR I'm open to naming suggestions here

Renamed because the trusted side has `stderr_write_all` which does the writing of the buffer, while this function redirects the stream on the host side.
Originally this was named `stderr_write_all` as I was thinking "set stderr_write_all" and I misrepresented the omitting of "get" in the [guidelines](https://rust-lang.github.io/api-guidelines/naming.html#getter-names-follow-rust-convention-c-getter) to omit "set"